src/sinks/aws_cloudwatch_metrics.rs

```diff
@@ -94,6 +94,10 @@ impl SinkConfig for CloudWatchMetricsSinkConfig {
     fn sink_type(&self) -> &'static str {
         "aws_cloudwatch_metrics"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
I think this should be true, no? I believe the `PartitionBatchSink` only acks after the events are sent?

Reply:
Right, if the underlying protocol supports it. ISTR some of rusoto interfaces don't make that information easily accessible. I was cribbing from the notes in the [`acknowledgement-reference-docs` branch](https://github.com/vectordotdev/vector/tree/acknowledment-reference-docs).